.github/workflows/workflow.yml

```diff
@@ -82,4 +82,4 @@ jobs:
         run: brew install shfmt
 
       - name: Run shfmt
-        run: make fmt-check
```

Comment:
Did this have to get renamed? I'm not opposed to the rename, but can we put it in a different PR just to keep this one small?

Reply:
Done: https://github.com/asdf-community/asdf-direnv/pull/173/files